web/src/popover_menus.js

```diff
@@ -918,4 +929,106 @@ export function initialize() {
             popover_instances.send_later = undefined;
         },
     });
+
+    register_popover_menu("#personal-menu", {
+        placement: "bottom",
+        offset: [-50, 0],
+        popperOptions: {
+            strategy: "fixed",
+            modifiers: [
+                {
+                    name: "eventListeners",
+                    options: {
+                        scroll: false,
+                        resize: false,
+                    },
+                },
+            ],
+        },
+        onMount(instance) {
+            const $popper = $(instance.popper);
+            $popper.addClass("personal-menu-tippy");
+            popover_instances.personal_menu = instance;
+            $popper.one("click", ".clear_status", (e) => {
+                e.preventDefault();
+                const me = page_params.user_id;
+                user_status.server_update_status({
```

Comment:
lets create a function `clear_status`, as its called twice now.

Reply:
@ryanreh99 I am not sure what you are trying to say here I was unable to find any code repetition here. Are you mentioning that there are 2 click handlers for `.clear_status`?

Can you please clarify more on this feedback?